README.md

```diff
@@ -1,44 +1,34 @@
-# Executor Router
+# Executor Queue
 [![Version][npm-image]][npm-url] ![Downloads][downloads-image] [![Build Status][status-image]][status-url] [![Open Issues][issues-image]][issues-url] [![Dependency Status][daviddm-image]][daviddm-url] ![License][license-image]
 
-> A generic executor plugin that routes builds to a specified executor
+> A generic executor plugin that routes builds through a queue
 
-An executor is an engine that is capable of running a set of docker containers together.
+An engine that will trigger a user's compute process to start and stop.
```

Comment:
I think it is missing "An Executor is ...", but I'm not sure this even needs to be explained here.